<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTail</class>
 <widget class="QDialog" name="NewTail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Aircraft</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLabel" name="operationLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Operation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="registrationLineEdit">
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>mandatory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="makeLineEdit">
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>e.g. Boeing</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="variantLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Variant</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="modelLineEdit">
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>e.g. 737</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="ppNumberComboBox">
     <property name="currentText">
      <string>&lt;Select&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>&lt;Select&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single Engine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multi Engine</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="weightLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Weight Class</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="weightComboBox">
     <item>
      <property name="text">
       <string>&lt;Select&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Light</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Heavy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Super</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="registrationLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="modelLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="companyLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="companyLineEdit">
     <property name="maxLength">
      <number>69</number>
     </property>
     <property name="placeholderText">
      <string>optional</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="maxLength">
      <number>40</number>
     </property>
     <property name="placeholderText">
      <string>start typing to search for aircraft types</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="variantLineEdit">
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>e.g. 800</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="searchLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="ppTypeComboBox">
     <property name="currentText">
      <string>&lt;Select&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>&lt;Select&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unpowered</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Piston</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turboprop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jet</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="powerPlantLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Power Plant</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="makeLabel">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Make</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="operationComboBox">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>&lt;Select&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>&lt;Select&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single Pilot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multi Pilot</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>registrationLineEdit</tabstop>
  <tabstop>companyLineEdit</tabstop>
  <tabstop>makeLineEdit</tabstop>
  <tabstop>modelLineEdit</tabstop>
  <tabstop>variantLineEdit</tabstop>
  <tabstop>operationComboBox</tabstop>
  <tabstop>ppTypeComboBox</tabstop>
  <tabstop>ppNumberComboBox</tabstop>
  <tabstop>weightComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewTail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>706</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
